tools/packaging/kata-deploy/local-build/dockerbuild/install_oras.sh

```diff
@@ -29,9 +29,21 @@ fi
 
 arch=$(uname -m)
 if [ "${arch}" = "ppc64le" ]; then
-	echo "An ORAS release for ppc64le is not available yet."
-	exit 0
-fi
+ 	echo "Building oras from source"
+	go_version="go1.21.3"
```

Comment:
Why don't you reuse the existing [script](https://github.com/kata-containers/kata-containers/blob/main/ci/install_go.sh) for this?

Reply:
Hmm... it fetches the `test` repo which is going to be deprecated soon. I think we have to rewrite the script in a separate PR and refactor this there. Just ignore this comment.

Reply:
@BbolroC, this is a temporary fix to build and use Oras, once the release it out for ppc64le, I will switch to using the artefacts from releases.